src/ext-test/java/org/opentripplanner/ext/fares/impl/FaresIntegrationTest.java

```diff
@@ -76,11 +76,11 @@ public void testPortland() {
       .toInstant();
 
     ItineraryFares fare = getFare(from, to, startTime, serverContext);
+    var fpu = List.copyOf(fare.getLegProducts().values()).getFirst();
+    assertEquals(List.of(fpu), List.copyOf(fare.getLegProducts().values()));
```

Comment:
What are we testing here? The size of `fare.getLegProducts().values()`?

Reply:
Ah you're right. This assertions makes no sense and I will fix it.